test/uk/gov/hmrc/nisp/views/LandingViewSpec.scala

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.nisp.views
+
+import org.scalatest.mock.MockitoSugar
+import play.api.mvc.{AnyContent, Request}
+import play.api.test.FakeRequest
+import play.api.test.Helpers.{contentAsString, _}
+import uk.gov.hmrc.http.HttpPost
+import uk.gov.hmrc.nisp.config.ApplicationConfig
+import uk.gov.hmrc.nisp.config.wiring.{NispFormPartialRetriever, WSHttp}
+import uk.gov.hmrc.nisp.controllers.FeedbackController
+import uk.gov.hmrc.nisp.controllers.auth.NispAuthedUser
+import uk.gov.hmrc.nisp.fixtures.NispAuthedUserFixture
+import uk.gov.hmrc.nisp.helpers._
+import uk.gov.hmrc.nisp.utils.{Constants, MockTemplateRenderer}
+import uk.gov.hmrc.nisp.views.html.landing
+import uk.gov.hmrc.play.partials.{CachedStaticHtmlPartialRetriever, FormPartialRetriever}
+import uk.gov.hmrc.renderer.TemplateRenderer
+
+class LandingViewSpec extends HtmlSpec with MockitoSugar {
+
+  val fakeRequest = FakeRequest("GET", "/")
+
+  val mockHttp = mock[WSHttp]
+
+  val testFeedbackController = new FeedbackController {
+    override implicit val cachedStaticHtmlPartialRetriever: CachedStaticHtmlPartialRetriever = MockCachedStaticHtmlPartialRetriever
+    override implicit val formPartialRetriever: FormPartialRetriever = MockFormPartialRetriever
+
+    override implicit val templateRenderer: TemplateRenderer = MockTemplateRenderer
+
+    override def httpPost: HttpPost = mockHttp
+
+    override def localSubmitUrl(implicit request: Request[AnyContent]): String = ""
+
+    override def contactFormReferer(implicit request: Request[AnyContent]): String = request.headers.get(REFERER).getOrElse("")
+
+    override val applicationConfig: ApplicationConfig = new ApplicationConfig {
```

Comment:
You can use test/uk/gov/hmrc/nisp/utils/FakeApplicationConfig.scala object instead